### PR TITLE
AV-192470 : ALB-SDK update to latest for AKO 1.12

### PR DESCRIPTION
**Pull request by DixitAakash**
This PR is to update the alb-sdk in AKO to the latest version.

### CODE REVIEW

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

File: pkg/utils/constants.go [hunk 1]
```diff
@@ -96,8 +96,6 @@ const (
 	VCF_CLUSTER                   = "VCF_CLUSTER"
 	MCI_ENABLED                   = "MCI_ENABLED"
 	USE_DEFAULT_SECRETS_ONLY      = "USE_DEFAULT_SECRETS_ONLY"
-	CTRL_VERSION_22_1_3           = "22.1.3"
-	CTRL_VERSION_22_1_2           = "22.1.2"
 	Namespace                     = "Namespace"
 	MaxAviVersion                 = "22.1.3"
```
Comment (arihantg) [inline pkg/utils/constants.go:100]: we should change MaxAviVersion also

**Comment by DixitAakash** [comment]
Closed the PR and will create a new one with the correct branch name.

**Pull request closed without merging**